### PR TITLE
Add multitenancy docs

### PR DESCRIPTION
**Pull request by Mark-Powers**

### CODE REVIEW

File: user/application_user.md [hunk 1]
```diff
@@ -30,11 +30,15 @@ It is important this container be built for ARM architecture, so that it can run
 
 If your service requires any types of peripherals, you can select these from a list when creating your service. Your service will only be able to run on devices with this peripheral type configured.
 
+If your service wants single-tenancy over any type of resource, select the type of resource. This metadata allows us to avoid scheduling applications at the same time that might be affected by the presence of others. For example, if you are collecting network bandwidth information, you will likely not want others to run bandwidth tests simultaneously, but it may be OK to run this at the same time another application reads from a sensor. If your application generally requires single-tenancy, you can enable this when submitting an "application."
```
Comment (marcwitasee) [inline user/application_user.md:33]: If this isn't defined elsewhere, can we add a sentence to clarify what we mean by resource?

File: user/application_user.md [hunk 2]
```diff
@@ -44,7 +48,7 @@ In addition to any environment variables defined here, and later in the job, we
 
 If your application requires secret values, such as S3 secrets, consider leaving them as blank. They can be overrode in a private job.
 
-A volume shared between your services will be mounted at `/share`.
+An ephemeral volume shared only between your services will be mounted at `/share`. Additionally, there is a shared, persistent volume mounted at `/public`. This directory is shared between all application services running on the device, and does not get automatically cleaned up.
```
Comment (marcwitasee) [inline user/application_user.md:51]: As opposed to the `/share` directory which is not shared between all applications and does get automatically cleaned up? I think a bit more clarify on the difference between these two volumes could be helpful.

**Review by marcwitasee — CHANGES_REQUESTED** [review]

File: user/application_user.md (hunk 2 above)
Comment (msherman64) [reply user/application_user.md:51]: So `/share` is automatically cleaned up when your job ends, and is only visible to containers within that job? What if "you" schedule multiple jobs onto the same device?

**Review by msherman64 — CHANGES_REQUESTED** [review]
I would rewrite the "volumes" section to be quite specific:

The volume mounted to `/share` in your pods is accessible read/write, and shared between all pods from the same Job. The underlying filesystem in this volume is located on the host device, and supports simultaneous read/write from multiple pods, sutiable for locking. When your job ends, the data in this path is deleted.

In contrast, the volume mounted to `/public/` is accessible as read/write to all pods on the device, not just from your Job. In the case of multi-tenancy, other users or projects may have access. The intended use of this volume is to "publish" data that will be read by other applications, for example a periodic uploader. The data in this volume is persistent, and will NOT be removed when you job ends. However, because of the multi-tenant access, you cannot rely on it to persist state between subsequent runs of an application in different Jobs.

File: user/application_user.md (hunk 1 above)
Comment (Mark-Powers) [reply user/application_user.md:33]: I'm removing this section, as I think it'll be unused for now, and is confusing.

**Review by marcwitasee — APPROVED** [review]

**Pull request merged**